app/src/main/res/layout-v21/layout_search.xml

```diff
@@ -97,27 +97,50 @@
                 android:layout_marginStart="8dp"
                 android:layout_marginTop="4dp"
                 android:layout_marginEnd="8dp"
-                android:layout_marginBottom="8dp"
                 app:chipSpacing="4dp"
                 app:singleLine="true" />
 
         </HorizontalScrollView>
 
-        <TextView
-            android:id="@+id/searchDeepSearchTV"
-            android:layout_width="wrap_content"
+        <LinearLayout
+            android:id="@+id/deepSearchContainer"
```

Comment:
I would prefer the same (https://github.com/TeamAmaze/AmazeFileManager/pull/4388/files#r2052791812) behaviour here too, but that's personal preference. let's wait for other maintainers to chime in :)

Reply:
You can open an issue and explain it in there, as I'm not quite sure how it wold look.